src/Question/question.controller.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  ApiBasicAuth,
+  ApiForbiddenResponse,
+  ApiOkResponse,
+  ApiQuery,
+  ApiTags,
+} from "@nestjs/swagger";
+import {
+  CacheInterceptor,
+  ClassSerializerInterceptor,
+  Controller,
+  Get,
+  UseInterceptors,
+  Query,
+  Param,
+} from "@nestjs/common";
+import { QumlQuestionService } from "src/adapters/diksha/quml.adapter";
```

Comment:
Can't this import by dynamic based on the value of `:adapter` ?

Reply:
sir dynamic import not possible based on value of adapter

Reply:
@ChakshuGautam @SaketSinha @arajput -- any inputs on how the adapter can be imported dynamically based on the adapter name from URL without any issues / changes in the bundling ?

Reply:
What would be the ideal import statement here?

Reply:
@ChakshuGautam  Ideal statement will be like 
`import { QumlQuestionService } from "src/adapters/${source}/quml.adapter";`

Reply:
@coolbung Till we get the solution on this, let's merge as per the conditional base use. @ChakshuGautam Please share your inputs on this. We will make changes as per it later.